### DISCUSSION TITLE
package doesn't have to be `var`

### DISCUSSION THREAD

**Discussion by mayoff** [pointfreeco/isowords#106] [General]
You mention in episode 144 that `package` (in `Package.swift`) is a `var`, not a `let`, so that you can mutate it later in the file. The `Package` type is a `class` with `var` properties, so you can mutate `package` even if you declare it `let`.

`Target` is also a `class` with `var` properties.

I personally found this surprising when I learned it, given the Swift team's strong preference for value types, so I wasn't sure if you were aware of it.

**Comment by mbrandonw** [comment]
lol we definitely were _not_ aware of that. I've added a [correction](https://github.com/pointfreeco/pointfreeco/commit/91a0778d651707b670045b4b9e709a26f8a516db) to the transcript that will go out with next deploy, and we'll update the `Package.swift` sometime soon.

Thanks for letting us know!
